config/config.exs

```diff
@@ -1,3 +1,4 @@
 import Config
 
 config :phoenix, :json_library, Jason
+config :heex_formatter, :tw_config_file, File.cwd!() <> "/config/tailwind_format.yml"
```

Comment:
I wonder if there is a better way to define the tailwind config file path. Also, it might be good to allow the devs to extract it and change it as they want, like we can do with the credo, for example, disabling - enabling checks we want